coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4859,3 +4859,28 @@ def forward(self, dividend, divisor):
                 return output_tensor
 
         self.run_compare_torch(shapes, RemainderModel(), backend=backend)
+
+
+class TestSum(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "backend, input_dtype",
+        itertools.product(
+            backends,
+            [np.int32, np.float32, np.bool]
+        )
+    )
+    def test_sum(self, backend, input_dtype):
+        model = ModuleWrapper(function=torch.sum)
+
+        input_data = torch.zeros(2, 3)
```

Comment:
With this input, I think the torch trace model are still going to have the torch input type as float32 ... 
I think, if you do `input_data = torch.zeros(2, 3).to(...)`, the internal torch IR graph are going to produce the input type as expected, so you don't need to explicitly pass `converter_input_type` ... 
And I think this is typically what a normal user will do ... especially 3rd party users they have limited knowledge about how to use `ct.TensorType(shape=input_data.shape, dtype=input_dtype)` 😆

Reply:
Make sense! Changed :)